### PR TITLE
feat: added translation for tooltips

### PR DESCRIPTION
**Pull request by pramodcog**
#### Type of change

Feat  

#### Jira ticket :blue_book:

https://cognitedata.atlassian.net/browse/

## Description :pencil:

Added translation support for all Tooltip in the `reveal-react-components`

## How has this been tested? :mag:

In storybook

## Test instructions :information_source:

In storybook, run Toolbar example, change language preference in browser setting to `Norsk` language. Reload the page and hover the mouse cursor on any button in the Toolbar.

## Checklist :ballot_box_with_check:

- [x] I am proud of this feature.
- [x] I have performed a self-review of my own code.
- [x] I have added PR type (Feat, Bug, Chore, Test, Docs, Style, Refactor) to the PR title.
- [ ] I have manually tested this for different scenarios (different models, formats, devices, browsers).
- [ ] I have commented my code in hard-to-understand areas.
- [ ] I have made corresponding changes to the public documentation.
- [ ] I have added unit and visuals tests to prove that my feature works to the best of my ability.
- [ ] I have refactored the code for readability to the best of my ability.
- [ ] I have checked that my changes do not introduce regressions in the public documentation.
- [ ] I have outlined any known defects / lacking capabilities in the contents of this PR.
- [ ] I have listed any remaining work that should follow this PR in the description or jira/miro/etc.
- [ ] I have added TSDoc to any public facing changes.
- [ ] I have added "developer documentation" in any package README.md that is related / applicable to this PR.
- [ ] I have noted down and am currently tracking any technical debt introduced in this PR.
- [ ] I have thoroughly thought about the architecture of this implementation.
- [ ] I have asked peers to test this feature.

### CODE REVIEW

File: react-components/src/common/i18n/en/reveal-react-components.json [hunk 1]
```diff
@@ -1,8 +1,11 @@
 {
   "CAD_MODELS": "CAD models",
+  "FIT_VIEW": "Fit view",
+  "HELP": "Help",
   "HIGH_FIDELITY": "High fidelity",
```
Comment (haakonflatval-cognite) [inline react-components/src/common/i18n/en/reveal-react-components.json:5]: Can we have `_TOOLTIP` suffix on all the keys?

**Review by haakonflatval-cognite — CHANGES_REQUESTED** [review]
Looks good! Just one comment

File: react-components/src/common/i18n/en/reveal-react-components.json (hunk 1 above)
Comment (pramodcog) [reply react-components/src/common/i18n/en/reveal-react-components.json:5]: I had the _TOOLTIP suffix (`IMAGES_360_DETAILS_TOOLTIP`) and realized that some the keys can be shared eg. `SETTINGS`, I can add them if you think it matters

Comment (pramodcog) [reply react-components/src/common/i18n/en/reveal-react-components.json]: Added

**Review by haakonflatval-cognite — APPROVED** [review]
Nice! LGTM :+1:

**Pull request merged**